packages/next-bundle-analyzer/index.d.ts

```diff
@@ -0,0 +1,3 @@
+import type { NextConfig } from "next";
```

Comment:
`next` is not a dependency here - is it safe to `import type` from it? 🤔

Reply:
Since this is only meant to be installed along with `next` sounds good